### PR TITLE
Added support for 2 UART ports

### PR DESCRIPTION
**Pull request by rahulbhivare**
Netra needs support for 2 UART ports.

### CODE REVIEW

File: examples/config.json [hunk 1]
```diff
@@ -21,7 +21,9 @@
             "GPIO_Interrupt": "0"
         },
         "UART": {
-            "Device_Path": "/dev/ttyUSB1"
+            "UART_PORT_COUNT": "2",
+            "Device_Path_0": "/dev/ttyUSB1",
+            "Device_Path_1": "/dev/ttyUSB1"
```
Comment (deepaktawri) [inline examples/config.json:26]: should this be ttyUSB2 ?

File: lib/python/satos_payload_sdk/antaris_api_gpio.py [hunk 2]
```diff
@@ -86,8 +92,16 @@ def api_pa_pc_get_gpio_pins_number(index):
     return value
 
 def api_pa_pc_get_uart_dev():
-    value = jsfile_data[g_JSON_Key_IO_Access][g_JSON_Key_UART][g_JSON_Key_Device_Path]
-    return value
+    g_total_uart_port = jsfile_data[g_JSON_Key_IO_Access][g_JSON_Key_UART][g_JSON_Key_Device_Count]
+    uart_dev = ["NULL" , "NULL"]
```
Comment (deepaktawri) [inline lib/python/satos_payload_sdk/antaris_api_gpio.py:96]: uart_dev size should be equal to g_total_uart_port ? If its two and g_total_uart_port is five, further code will be in trouble.

**Review by deepaktawri — CHANGES_REQUESTED** [review]

File: examples/config.json (hunk 1 above)
Comment (rahulbhivare) [reply examples/config.json:26]: Updated

File: lib/python/satos_payload_sdk/antaris_api_gpio.py (hunk 2 above)
Comment (rahulbhivare) [reply lib/python/satos_payload_sdk/antaris_api_gpio.py]: Updated library

**Review by deepaktawri — APPROVED** [review]

**Review by bcwaldon — APPROVED** [review]

**Pull request merged**